beep/featurize.py

```diff
@@ -718,16 +866,10 @@ def features_from_processed_cycler_run(cls, processed_cycler_run, params_dict=No
         Qc10_1 = diagnostic_interpolated.charge_capacity[
                  start_list[params_dict["cycle_comp_num"][0]]:
                  start_list[params_dict["cycle_comp_num"][0]] + params_dict["Q_seg"]]
```

Comment:
Could you add a TODO here? I realized that the `"Q_seg"` value is really just the number of interpolated datapoints for these diagnostic cycles so if we ever change that, this value will also need to be changed.

Reply:
done